Resources/Prototypes/_Floof/Traits/marked.yml

```
@@ -0,0 +1,21 @@
+- type: trait
+  id: TraitorKillTarget
+  category: Marked
+  points: 0
+  slots: 0
+  functions:
+  - !type:TraitAddComponent
+    components:
+    - type: Marked
+      targetType: TraitorKill
+
+- type: trait
+  id: TraitorTeachTarget
+  category: Marked
+  points: 0
+  slots: 0
```

Just noticed those don't give any points. I think it would make perfect sense for them to give something to incentivise people to take them for reasons other than boredom. Not too much, because you are not guaranteed to be a target, so something like:
```suggestion
  points: 2
  slots: 0
  functions:
  - !type:TraitAddComponent
    components:
    - type: Marked
      targetType: TraitorKill

- type: trait
  id: TraitorTeachTarget
  category: Marked
  points: 1
  slots: 0
```

---

I had set those to 0 at the beginning with the intention to ask about how much people thought they'd be worth and then promptly forgot about them. 1 and 2 seems good and they can always be easily adjusted up or down based on community feedback.